src/templates/landing-page.js

```diff
@@ -1,33 +1,66 @@
-import React from "react";
+import React, { useEffect } from "react";
 // import React, { Suspense } from "react";
-import { graphql } from "gatsby";
-import { I18nextProvider, useTranslation } from "react-i18next";
+// import { Redirect, Router } from "@reach/router";
 import { Helmet } from "react-helmet";
+import { graphql, navigate } from "gatsby";
+import { I18nextProvider, useTranslation } from "react-i18next";
 import { GridContainer } from "@trussworks/react-uswds";
 // import { Graphic, Hero, i18n, Layout, Section, Tagline } from "../components/index";
 
 import Graphic from "../components/graphic";
 import Hero from "../components/hero";
-import i18n from "../components/i18n";
+import i18next from "../components/i18n";
 import Layout from "../components/layout";
 import Section from "../components/section";
 import Tagline from "../components/tagline";
+// import NotFound from "../components/404";
 
 import "@trussworks/react-uswds/lib/uswds.css";
 import "@trussworks/react-uswds/lib/index.css";
 
-function Landing({ data }) {
+function Landing({ data, location }) {
+  const { t, i18n } = useTranslation();
+
+  useEffect(() => {
+    const path = location.pathname;
+    const lang = path.split("/")[1];
+    const route = path
+      .split("/")
+      .slice(2)
+      .filter((v) => v != "")
+      .join("/");
+
+    if (lang.length === 0 && route.length === 0) {
```

Comment:
Thanks for finding a way to change language on the url change! I wonder though if we can utilize language  detector or another i18next feature?

Reply:
Hmm, from what I have seen in the documentation before, I don't think so. Does anything stick out to you?

Reply:
I'll try to look into the docs more, but when I looked a week or two ago I didn't find any good examples